Sources/MapboxMaps/Gestures/GestureManager+GestureHandlerDelegate.swift

```diff
@@ -35,7 +35,7 @@ extension GestureManager: GestureHandlerDelegate {
     // MapView has been panned
     internal func panned(from startPoint: CGPoint, to endPoint: CGPoint) {
         if let cameraOptions = cameraManager.mapView?.mapboxMap.dragCameraOptions(from: startPoint, to: endPoint) {
-            cameraManager.setCamera(to: cameraOptions)
+            cameraManager.mapView?.mapboxMap.setCamera(to: cameraOptions)
```

Comment:
Why are we not guarding this option mapViews?
Under what condition would `mapView` be nil here? This would cause a potential run time crash.